db/models.py

```diff
@@ -50,3 +53,54 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [UniqueConstraint(
+            fields=["row", "seat", "movie_session"],
+            name="unique_movie_hall"
+        )]
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie} "
+                f"{self.movie_session.show_time} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        if self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                "row": [f"row number must be in available range: "
+                        f"(1, rows): "
+                        f"(1, {self.movie_session.cinema_hall.rows})"]
+            })
+        if self.seat > self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError({
+                "seat": [f"seat number must be in available range: "
+                         f"(1, seats_in_row): "
+                         f"(1, {self.movie_session.cinema_hall.seats_in_row})"]
+            })
+
+    def save(self, *args, **kwargs) -> super:
+        self.full_clean()
+        return super().save(*args, **kwargs)
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model inherits from `AbstractUser`, which is the correct approach for extending Django's default user model while maintaining compatibility with its authentication system.